modules/creating-serverless-apps-kn.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// serverless/serving-creating-managing-apps.adoc
+
+[id="creating-serverless-apps-kn_{context}"]
+= Creating serverless applications using the kn CLI
+
+.Prerequisites
+* Installed the {ServerlessOperatorName} and Knative Serving.
+* Installed `kn` CLI.
+
+.Procedure
+
+. Create the Knative service by entering:
++
+----
+$ kn service create <SERVICE-NAME> --image <IMAGE-URL> --env <ENVIRONMENT-VARIABLE-NAME=VALUE>
```

Comment:
Image names are not URLs. , so maybe just <IMAGE> ,  also the "ENVIRONMENT-VARIABLE-NAME=VALUE" looks a bit confusing to me with all the dashes... 

FWIW, the `kn service create --help`

describes it as

> Usage:
>   kn service create NAME --image IMAGE [flags]

with multiple examples, among them

>   Create a service with multiple environment variables
>   kn service create mysvc --env KEY1=VALUE1 --env KEY2=VALUE2 --image dev.local/ns/image:latest

Reply:
updated it just to `KEY=VALUE` but I'm not sure if this covers what you mean, lmk if that's OK